### PR TITLE
feat: Added Navigation component

### PR DESCRIPTION
**Pull request by arjunbharti**
Created navigation component with _Title, contact and some links._
Preview: https://deploy-preview-6--uivibe.netlify.app/documentation/

### CODE REVIEW

File: components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.navbar-container{
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    padding: 1.5rem 2.5rem;
+    max-height: 3.75rem;
+    width: 100%;
+    align-items: center;
+    box-shadow: 0 4px 4px -6px;
```
Comment (darshandamre) [inline components/navigation.css]: you haven't given a color to box-shadow

File: components/navigation.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.navbar-container{
+    display: flex;
+    justify-content: space-between;
+    box-sizing: border-box;
+    padding: 1.5rem 2.5rem;
+    max-height: 3.75rem;
+    width: 100%;
+    align-items: center;
```
Comment (darshandamre) [inline components/navigation.css:8]: one suggestion here, have all the flex related properties grouped together, helps to understand what all things flex is doing to a element at a glance

**Review by darshandamre — COMMENTED** [review]
one more suggestion keep updating the links in sidebar helps navigate the components easily

File: components/navigation.css (hunk 2 above)
Comment (arjunbharti) [reply components/navigation.css:8]: Will push the changes at the end else there will be so many merge conflicts

File: components/navigation.css (hunk 1 above)
Comment (arjunbharti) [reply components/navigation.css]: Thanks for pointing it out. Updated successfully

**Review by sameep-s — APPROVED** [review]
You are good to go.

**Pull request merged**